modules/odc-viewing-application-topology.adoc

```diff
@@ -5,9 +5,14 @@
 [id="odc-viewing-application-topology_{context}"]
 = Viewing the topology of your application
 
-You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology* view where you can see the status of the application Pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build.
+You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology* view where you can see the status of the application Pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
 
-A serverless application is visually indicated with the Knative symbol (image:odc_serverless_app.png[title="Serverless Application"]). You can zoom in and out to see more details for a particular application.
+A serverless application is visually indicated with the Knative symbol (image:odc_serverless_app.png[title="Serverless Application"]).
+
+[NOTE]
+====
+Serverless applications take some time to load and display on the *Topology* view. When you create a serverless application, it first creates a service resource and then a revision, after which it is deployed and displayed on the Topology view. If it is the only workload, you may be redirected to the *Add* page. Once the revision is deployed, the serverless application is displayed on the *Topology* view.
```

Comment:
service resource, and then a revision. After that, it is deployed and displayed in the *Topology* view.

s/may be/might be